index.Rmd

```diff
@@ -169,9 +169,9 @@ The book covers a wide range of topics and will be of interest to a wide range o
 
 > With the advent of 'modern' GIS software, most people want to point and click their way through life. That’s good, but there is a tremendous amount of flexibility and power waiting for you with the command line.
 
-- Graduate students and researchers from fields specializing in geographic data including Geography, Remote Sensing, Planning, GIS and Geographic Data Science
-- Academics and post-graduate students working on projects in fields including Geology, Regional Science, Biology and Ecology, Agricultural Sciences (precision farming), Archaeology, Epidemiology, Transport Modeling, and broadly defined Data Science which require the power and flexibility of R for their research <!-- please add further fields-->
-- Applied researchers and analysts in public, private or third-sector organizations who need the reproducibility, speed and flexibility of a command-line language such as R in applications dealing with spatial data as diverse as Urban and Transport Planning, Logistics, Geo-marketing (store location analysis) and Emergency Planning <!-- please add further examples-->
+- Graduate students and researchers from fields specializing in geographic data which includes Geography, Remote Sensing, Planning, GIS and Geographic Data Science
+- Academics and post-graduate students who are working on projects in fields like Geology, Regional Science, Biology and Ecology, Agricultural Sciences (precision farming), Archaeology, Epidemiology, Transport Modeling, Medical Sciences, Economical surveys, and broadly defined Data Science which require the power and flexibility of R for their research <!-- please add further fields-->
+- Applied researchers and analysts in public, private or third-sector organizations who need the reproducibility, speed and flexibility of a command-line language such as R in applications dealing with spatial data as diverse as Urban and Transport Planning, Logistics, Geovisualization, Network Analysis, Spatial descriptive statistics, Geo-marketing (store location analysis) and Emergency Planning <!-- please add further examples-->
```

Comment:
Looking good but 'Spatial descriptive statistics' and 'Geovisualisation' are not applied fields. I think it was OK before.